solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudHttp2SolrClient.java

```diff
@@ -147,6 +148,7 @@ public static class Builder {
     private ResponseParser responseParser;
     private long retryExpiryTime =
         TimeUnit.NANOSECONDS.convert(3, TimeUnit.SECONDS); // 3 seconds or 3 million nanos
+    private int parallelCacheRefreshesLocks = 3;
```

Comment:
what about 
```
private int parallelCacheRefreshesLocks = Integer.getInteger("solr.cloudSolrClient.parallelCacheRefreshLocks", 3);
```
this will allow setting via the system property and also builder